tests/test_cli.py

```diff
@@ -115,6 +115,19 @@
             ),
             "git+ssh://git@gitlab.com/akibahmed/sops-secrects.git?ref=main&shallow=1",
         ),
+        (
+            Node.from_dict(
+                {
+                    "original": {
+                        "type": "sourcehut",
+                        "owner": "~rycee",
+                        "repo": "nmd",
+                        "url": "sourcehut:~rycee/nmd",
```

Comment:
doesn't look like it has URL when i searched github for examples
```
      },
      "original": {
        "owner": "~tomterl",
        "repo": "fullframe",
        "type": "sourcehut"
      }
```
https://github.com/emacs-twist/examples/blob/a6e840259b545cc4083def36665b8e30bc781827/profiles/terlar/lock/flake.lock#L1391